NuGet.config

```diff
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
```

Comment:
Revert the change to this file.  This PR will not be accepted until this is completed.  Just checkout the previous version of the file, please.